source/cursorManager.py

```diff
@@ -179,7 +179,12 @@ def doFindText(self, text, reverse=False, caseSensitive=False, willSayAllResume=
 			if not willSayAllResume:
 				speech.speakTextInfo(info, reason=controlTypes.OutputReason.CARET)
 		else:
-			wx.CallAfter(gui.messageBox,_('text "%s" not found')%text,_("Find Error"),wx.OK|wx.ICON_ERROR)
+			wx.CallAfter(
+				gui.messageBox,
+				_('text "%s" not found') % text,

```

Comment:
Add translation comment

```suggestion
				# Translators: message displayed to the user when
				# searching text and no text is found.
				_('text "%s" not found') % text,
```